README.md

```diff
@@ -1,65 +1,51 @@
-# Base template for Gulp
+# Calendar table
 
-Template to start from before starting frontend projects.
+## Task
 
-## Install template
+Create a function that generates a calendar for a given pair 
+(year, month).
 
-- Create empty repository for your project. [learn how](#) _need a link here_
-- Clone boilerplate into project directory and navigate there.
-```bash
-$ git clone https://github.com/mate-academy/gulp-template.git project-name
-$ cd project-name
-```
-- Change remote repository to recently created
-```bash
-$ git remote set-url origin github-link-of-project
-```
-- Install dependencies
-```bash
-$ npm install
-```
+Function `calendarTable(year, month, element)`
 
-All packages will be installed after executing the script above.
+`element` - it's a DOM element.
 
-## Development mode 
-
-```bash
-$ npm run start
-```
+The calendar must be a table with `thead` and `tbody`, where day it's `TD`.
```

Comment:
I would change the description like this:

Create a function generating HTML `table` (with `thead` and `tbody`) inside a given`element` for given `year` and `month`

Reply:
done